code/modules/antagonists/vampire/vampire_powers/hemomancer_powers.dm

```diff
@@ -155,6 +243,36 @@
 	H.required_blood = 50
 	return H
 
+/obj/effect/proc_holder/spell/vampire/predator_senses
+	name = "Predator Senses"
+	desc = "Hunt down your prey, there's nowhere to hide..."
+	gain_desc = "Your senses are hightened, nobody can hide from you now."
+	action_icon_state = "predator_sense"
+	base_cooldown = 20 SECONDS
+
+/obj/effect/proc_holder/spell/vampire/predator_senses/create_new_targeting()
+	var/datum/spell_targeting/alive_mob_list/A = new()
+	A.allowed_type = /mob/living/carbon/human
+	A.max_targets = 300 // hopefully we never hit this number
+	return A
+
+/obj/effect/proc_holder/spell/vampire/predator_senses/valid_target(mob/target, mob/user)
+	return target.z == user.z && target.mind
+
+/obj/effect/proc_holder/spell/vampire/predator_senses/cast(list/targets, mob/user)
+	var/targets_by_name = list()
+	for(var/mob/living/carbon/human/H as anything in targets)
+		targets_by_name[H.real_name] = H
+
+	var/target_name = input(user, "Person to Locate", "Blood Stench") in targets_by_name
+	if(!target_name)
+		return
+	var/mob/living/carbon/human/target = targets_by_name[target_name]
+	var/message = "[target_name] is in [get_area(target)], [dir2text(get_dir(user, target))] from you."
+	if(target.get_damage_amount() >= 40 || target.bleed_rate)
+		message += " They are wounded..."
```

Comment:
I like this, but would it maybe make sense to make it italic, just to distinguish it as something distinct?